### ISSUE TITLE
Don't include 'enable_only=sssd' in the localauth plugin config

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#2923]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/2920
- Created at 2016-01-11 16:13:56 by [jhrozek](https://pagure.io/user/jhrozek)
- Closed as Duplicate
- Assigned to nobody
---

This came up during an IRC discussion with Sumit as a cheap way to proceed towards fixing ticket #2707.

If we drop 'enable_only = sssd' from localauth_plugin. Then the krb5 default should be used after the SSSD plugin does not give a definitive result.

### Comments

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2016-01-12 14:28:10](https://pagure.io/SSSD/sssd/issue/2920#225103)**

Is this ticket duplicate of #2788?

cc:  => sbose, lslebodn

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2016-01-12 15:24:26](https://pagure.io/SSSD/sssd/issue/2920#225104)**

Replying to [comment:1 lslebodn]:
> Is this ticket duplicate of #2788?

yes, since root is not handled by SSSD it is expected that the localauth plugin will return KRB5_PLUGIN_NO_HANDLE. With 'enable_only = sssd' removed libkrb5 should fallback to the default methods which include .k5login checks.

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2016-01-14 16:38:34](https://pagure.io/SSSD/sssd/issue/2920#225105)**

Closing as a duplicate, then

resolution:  => duplicate
status: new => closed

---

**Comment from [jhrozek](https://pagure.io/user/jhrozek) at [2017-02-24 14:31:07](https://pagure.io/SSSD/sssd/issue/2920#225106)**

**Metadata Update from @jhrozek**:
- Issue marked as blocked by: #2707
- Issue set to the milestone: NEEDS_TRIAGE
